.moban.yml

```diff
@@ -0,0 +1,9 @@
+configuration:
+  template_dir:
+    - .moban.dt/
+    - ../coala-mobans/templates/
+  configuration: devops.yml
+  configuration_dir: ../coala-mobans/
+targets:
+  - "nginx/nginx.conf.d/coala.io.conf": "coala.io.conf.jj2"
```

Comment:
we need quotes because ... ?  the `/` ?

Reply:
no, i think i was looking at configs on pyexcel. let me change that.